packages/evm/src/web2login.ts

```diff
@@ -53,3 +61,47 @@ export const connect = async (
     type: DappRequestType.EvmWeb2Login,
   })
 }
+
+export const connectWithRedirect = (
+  redirectURL: string,
```

Comment:
Did you forget to a add the optional `uid` parameter and pass `uid` into `request` as `signerAddress`? Once `uid` is provided, JoyID will prompt the user either to create an account or to login with an existing one, then sign and returning the entropy.

Reply:
see:
https://github.com/nervina-labs/joyid-sdk-js/blob/main/packages/evm/src/web2login.ts#L30

https://docs.joyid.dev/guide/evm/web2-login#connect-to-joyid